src/components/news/content/11/body.md

```diff
@@ -0,0 +1,7 @@
+Game* (Game Star) vise à rassembler les actrices et acteurs du jeu vidéo sur le campus de l'UNIL/EPFL. Ses activités sont variées, mais cette commission a notamment participé à l'organisation et organisé de beaux événements, tels que l'eSpace Arlaud lors des Jeux Olympiques de la Jeunesse, Games on Campus, ou encore Game* Speedrun Fest. Une autre de ses missions est d'informer les étudiantes et étudiants sur les possibilités liées au jeu vidéo sur le campus, tant au niveau associatif qu'académique - car des cours sur le jeu vidéo existent !
```

Comment:
Does the stars (`*`) need to be escaped? In GH it render as italic, will it in the website too?

Reply:
No, the spec requires that there are no spaces between the opening `*` and the first word of the italic bit. GFM is more aggressive in general to allow faster typing, but I've tested the change locally.